### ISSUE TITLE
Mark words from built-in dictionary to be flagged

### ISSUE THREAD

**Issue by MeAmAnUsername** [streetsidesoftware/vscode-spell-checker#1498]
There are a few English words that I do not want to use in my document. It is a formal document, so no contractions like `aren't`, `we've`. I also keep mixing up `therefore` ("because of") and `therefor` ("in front of there"), so I'd like to flag `therefor`.
How exactly should I mark words from the built-in dictionary to be flagged?

@Jason3S provides instructions in https://github.com/streetsidesoftware/vscode-spell-checker/issues/117#issuecomment-782184579 (and those same instructions are in the README on the main page) but they do not work for me.

I do not have a `cSpell.json` in the root directory of my project.
I have the following `.vscode/cSpell.json`:
```
{
    "version": "0.1",
    "language": "en",
    "words": [
        <omitted for brevity>
    ],
    "flagWords": [
        "aren't",
        <omitted for brevity>
        "therefor"
    ]
}
```

But `Test aren't test therefor test test` is not flagged.

It still is not flagged after adding
```
"cSpell.flagWords": [
    "therefor",
    "aren't"
]
```
to `.vscode/settings.json`. That section is darker than the rest of that file, so I guess that means those settings are ignored.

It does flag `tesqfd`, so the extension is active.

**Comment by Jason3S** [comment]
@MeAmAnUsername,

`flagWords` is the correct setting.

![image](https://user-images.githubusercontent.com/3740137/140152873-2cb8ee0f-d0e2-43ed-803d-002484565546.png)

Using the VS Code Preferences UI:
![image](https://user-images.githubusercontent.com/3740137/140155601-9fe79dff-d351-409d-924a-1a05c79bda18.png)

Being darker than the rest seem concerning. Please me sure the `settings.json` file is valid. The same thing for the `.vscode/cSpell.json` file.

**Comment by Jason3S** [comment]
You can also try running:

```sh
npx cspell trace therefor
```

It will show you if it is considered flagged.

![image](https://user-images.githubusercontent.com/3740137/140162959-626c6e71-b7b7-47b1-aaf0-1999ed913238.png)

**Comment by MeAmAnUsername** [comment]
Turns out I'm using Spell Right instead of Code Spell Checker, but when I googled "spell right vscode remove word from dictionary" it showed this repository as the first result.
I don't know where the cSpell file came from.

Thanks for the quick replies.

**Comment by MeAmAnUsername** [comment]
I switched to Spell Checker because Spell Right cannot flag words from the dictionary.
Spell Checker does not flag everything here:
![image](https://user-images.githubusercontent.com/15816455/140377611-2640d893-241c-4872-b077-0606a16841d3.png)

I would expect it to flag everything except `Test` and `test`. Does it split on `'` and not flag words shorted than 2 letters?

My `.vscode/cSpell.json`:
```
{
    "version": "0.2",
    "language": "en",
    "words": [
        <omitted>
    ],
    "flagWords": [
        <omitted>
        "therefor",
        <omitted>
        "they'd",
        "they'll",
        "they're",
        "they've",
        <omitted>
        "therefor"
    ]
}
```

```
MBP:dir user$ npx cspell trace therefor
bash: npx: command not found
```
Would this work even if I did install nodeJS? Where does it get the cspell package?

**Comment by Jason3S** [comment]
@MeAmAnUsername,

There seems to be an issue when part of the word is considered correct. I opened a new issue with the library that does the checking.

It is strange that Node isn't already installed on your MBP. Even so, you would need Node and NPM to run the `npx` command. Node 16 should work just fine.

**Comment by Jason3S** [comment]
I'm going to close this issue, because it is possible to mark words as forbidden except if they are made of parts too short for it to care. I opened a new issue with the library to take care of that.
